<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AoideTrackSearchOverlayFilterDlg</class>
 <widget class="QDialog" name="AoideTrackSearchOverlayFilterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>aoide Track Search Overlay Filter</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>262</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutSession">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutSessionStart">
        <item>
         <widget class="QLabel" name="sessionStartLabel">
          <property name="text">
           <string>Session start</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignVCenter">
         <widget class="QDateTimeEdit" name="sessionStartInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="sessionRestartButton">
          <property name="text">
           <string>Restart</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutBpm">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutMinBpm">
        <item>
         <widget class="QLabel" name="minBpmLabel">
          <property name="text">
           <string>Min. BPM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="minBpmDoubleSpinBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="minBpmResetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutMaxBpm">
        <item>
         <widget class="QLabel" name="maxBpmLabel">
          <property name="text">
           <string>Max. BPM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="maxBpmDoubleSpinBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="maxBpmResetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="genreHorizontalLayout">
      <item>
       <widget class="QLabel" name="genreTextLabel">
        <property name="text">
         <string>Genres (any, split at ';')</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="genreTextLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hashtagAllHorizontalLayout">
      <item>
       <widget class="QLabel" name="hashtagAllTextLabel">
        <property name="text">
         <string>Hashtags (all, no #, split at whitespace)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hashtagAllTextLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="commentAllHorizontalLayout">
      <item>
       <widget class="QLabel" name="commentAllTextLabel">
        <property name="text">
         <string>Comment terms (all, split at whitespace)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commentAllTextLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="commentAnyHorizontalLayout">
      <item>
       <widget class="QLabel" name="commentAnyTextLabel">
        <property name="text">
         <string>Comment terms (any, split at whitespace)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commentAnyTextLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AoideTrackSearchOverlayFilterDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AoideTrackSearchOverlayFilterDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
